lib/connections/service.go

```diff
@@ -758,7 +759,7 @@ func dialParallel(deviceID protocol.DeviceID, dialTargets []dialTarget) (interna
 	for _, prio := range priorities {
 		tgts := dialTargetBuckets[prio]
 		res := make(chan internalConn, len(tgts))
-		wg := sync.NewWaitGroup()
+		wg := stdsync.WaitGroup{}
```

Comment:
What is this about?

Reply:
our sync package is used for debugging long lasting locks/wg's. In this case the wg is expected to take time as it waits up to 10s to timeout.